src/generic/create-or-rerun-course/CreateOrRerunCourseForm.jsx

```diff
@@ -204,21 +204,20 @@ const CreateOrRerunCourseForm = ({
                 ref={field?.ref}
               />
             ) : (
-              <Form.Autosuggest
-                value={values[field.name]}
+              <TypeaheadDropdown
+                readOnly={false}
                 name={field.name}
+                value={field.value?.toLowerCase()}
+                controlClassName={classNames({ 'is-invalid': hasErrorField(field.name) })}
+                options={field.options}
                 placeholder={field.placeholder}
-                onSelected={(value) => setFieldValue(field.name, value)}
-                onBlur={handleCustomBlurForDropdown}
-                isInvalid={hasErrorField(field.name)}
-                key={uuid()}
-              >
-                {field?.options?.map((option) => (
-                  <Form.AutosuggestOption key={option}>
-                    {option}
-                  </Form.AutosuggestOption>
-                ))}
-              </Form.Autosuggest>
+                handleBlur={handleCustomBlurForDropdown}
+                handleChange={(value) => setFieldValue(field.name, value)}
+                noOptionsMessage={intl.formatMessage(messages.courseOrgNoOptions)}
+                helpMessage=""
+                errorMessage=""
+                floatingLabel=""
```

Comment:
Can we make optional props for this component? Add change to @edx/frontend-lib-content-components

Reply:
I think this task would take a lot of effort, it's better to leave it as it is.